pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -160,6 +163,17 @@ type FilterPlugin interface {
 	Filter(pc *PluginContext, pod *v1.Pod, nodeName string) *Status
 }
 
+// PostFilterPlugin is an interface for Post-filter plugin. Post-filter is an
+// informational extension point. Plugins will be called with a list of nodes
+// that passed the filtering phase. A plugin may use this data to update internal
+// state or to generate logs/metrics.
+type PostFilterPlugin interface {
+	Plugin
+	// PostFilter is called by the scheduling framework after a list of nodes
+	// passed the filtering phase.
+	PostFilter(pc *PluginContext, pod *v1.Pod, nodes []*v1.Node, nodeToStatuses NodeToStatusesMap) *Status
```

Comment:
might be expensive to pass the map by value, should we just pass a pointer? also I would name the variable filteredNodesStatuses.

Reply:
> might be expensive to pass the map by value, should we just pass a pointer? 

No, it's not. A map is a pointer itself, passing a pointer to the map or a map make no difference here.

Reply:
SG, I keep confusing C++ with golang!